app/src/sponsors/migrations/0002_sponsor_sponsor_type.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.16 on 2022-11-29 10:20
```

Comment:
You could remove both new migration files (i.e. 0002 and 0003) and re-run making migrations to have a single file.